uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/BAO/Document.php

```diff
@@ -11,7 +11,7 @@ class CRM_Tasksassignments_BAO_Document extends CRM_Tasksassignments_DAO_Documen
    * Create a new Document based on array-data
    *
    * @param array $params key-value pairs
-   * @return CRM_Tasksassignments_DAO_Document|NULL
```

Comment:
I can delete it, just the IDE doesn't know that $instance is a CRM_Tasksassignments_DAO_Document so I added it to hide the warning that it was returning something unexpected. The parent class also has a return which includes `object`.

![image](https://cloud.githubusercontent.com/assets/6374064/25375758/b60f91f2-2999-11e7-9c6a-a11c8695e2c4.png)